spark-operator/base/kustomization.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+# Labels to add to all resources and selectors.
+commonLabels:
+  kustomize.component: spark-operator
+  app.kubernetes.io/instance: spark-operator
+  app.kubernetes.io/managed-by: Tiller
+  app.kubernetes.io/name: sparkoperator
+  helm.sh/chart: sparkoperator-0.2.4
+
+# Images modify the tags for images without
+# creating patches.
+images:
+- name: gcr.io/spark-operator/spark-operator
+  newTag: v2.4.0-v1beta1-0.8.2
+
+# Value of this field is prepended to the
+# names of all resources
+namePrefix: spark-operator-spark
+
+# List of resource files that kustomize reads, modifies
+# and emits as a YAML string
+resources:
+- spark-operator-spark-sa.yaml
+- spark-operator-sparkoperator-cr-clusterrole.yaml
```

Comment:
Do we need to repeat "sparkoperator" in the file name here? The directory name already contains spark-operator, so I think these can all be omitted.

You can checkout the Pytorch one as an example: https://github.com/kubeflow/manifests/tree/master/pytorch-job

Also note that most of the manifests follow a 2-level structure - top level for "package" and lower level for "components".